Index: Library/TipoFuego.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>namespace DefaultNamespace;\n\nnamespace Pokemones_personal;\n\npublic class TipoFuego : ITipo\n{\n    public string Nombre { get; } = \"Fuego\";\n\n    public double CalcularAtaqueEfectivo(ITipo tipoAtaque)\n    {\n        if (tipoAtaque is TipoAgua or TipoAire)\n        {\n            return 0.5;  // Fuego es débil contra Agua.\n        }\n\n        if (tipoAtaque is TipoTierra)\n        {\n            return 1.25;  // Fuego tiene una ligera ventaja sobre Tierra.\n        }\n        \n        return 1.0;  // Neutral si no se cumple ninguna de las condiciones anteriores.\n    }\n}
===================================================================
diff --git a/Library/TipoFuego.cs b/Library/TipoFuego.cs
--- a/Library/TipoFuego.cs	
+++ b/Library/TipoFuego.cs	
@@ -1,7 +1,5 @@
 namespace DefaultNamespace;
 
-namespace Pokemones_personal;
-
 public class TipoFuego : ITipo
 {
     public string Nombre { get; } = "Fuego";
